src/entities/direct-thread.entity.ts

```diff
@@ -41,6 +45,31 @@ export class DirectThreadEntity extends Entity {
       },
     });
   }
+
+  public async updateTitle(title: string): Promise<DirectThreadRepositoryUpdateTitleResponseRootObject> {
```

Comment:
It is not necessary to specify function type if TS compilator can unambiguously define it from `return` statement.
In this case  `this.client.directThread.updateTitle` type is already defined, so TypeScript will know type of `DirectThreadEntity.broadcastLink`, no need to duplicate it twice.